src/lighthouse.php

```diff
@@ -305,9 +307,14 @@
             ],
             'pusher' => [
                 'driver' => 'pusher',
-                'routes' => \Nuwave\Lighthouse\Subscriptions\SubscriptionRouter::class.'@pusher',
+                'routes' => SubscriptionRouter::class . '@pusher',
                 'connection' => 'pusher',
             ],
+            'redis' => [
```

Comment:
Maybe this should be `echo` instead? Looking at https://github.com/tlaverdure/laravel-echo-server#subscribers, it seems like using Redis is just one possible method of broadcasting through Echo.